### PR TITLE
Use correct cms service

### PR DESCRIPTION
**Pull request by soniaklimas**
I want to merge this change because allows to switching cms service through Vercel env.

# Pull Request Checklist

- [ ] Test the changes locally to ensure they work as expected.
- [ ] Document the testing process and results in the pull request description. (Screen recording, screenshot etc)
- [ ] Include new tests for any new functionality or significant changes.
- [ ] Ensure that tests cover edge cases and potential failure points.
- [ ] Document the impact of the changes on the system, including potential risks and benefits.
- [ ] Provide a rollback plan in case the changes introduce critical issues.
- [ ] Update documentation to reflect any changes in functionality.

### CODE REVIEW

File: apps/storefront/src/services/cms.ts [hunk 1]
```diff
@@ -1,28 +1,51 @@
...
+    return butterCMSMenuService({
+      token: clientEnvs.NEXT_PUBLIC_BUTTER_CMS_API_KEY,
+    });
+  }
+};
+
+export const cmsPageServicePromise: Promise<CMSPageService> =
+  getCMSPageService();
+export const cmsMenuServicePromise: Promise<CMSMenuService> =
+  getCMSMenuService();
```
Comment (karolkarolka) [inline apps/storefront/src/services/cms.ts:51]: should work and no awaiting would be needed when importing the services in other files
```suggestion
export const cmsPageService: CMSPageService = await getCMSPageService();
export const cmsMenuService: CMSMenuService = await getCMSMenuService();
```

**Review by karolkarolka — CHANGES_REQUESTED** [review]

**Review by karolkarolka — APPROVED** [review]
Check before merging if setting CMS_SERVICE env on Vercel works on for your Preview build

**Comment by soniaklimas** [comment]
I tested it and it works correctly - if you change CMS_SERVICE env on Vercel and do redeploy chosen CMS is used.
